src/components/DataExplorer/DataExplorerInput.js

```diff
@@ -73,6 +73,18 @@ export default class DataExplorerInput extends React.Component {
     clearTimeout(this.inputAnimationTimeout)
   }
 
+  isNotAggSeed(subject) {
```

Comment:
Usually you want to make these "affirmative" so `isAggSeed` then if you want to negate them just do `!isAggSeed` I find it much easier to read